examples/MomentoLoadGen/Program.cs

```diff
@@ -0,0 +1,394 @@
+                AsyncSetGetResult.RESOURCE_EXHAUSTED => Interlocked.Increment(ref context.GlobalResourceExhaustedCount),
+                AsyncSetGetResult.RST_STREAM => Interlocked.Increment(ref context.GlobalRstStreamCount),
+                _ => throw new Exception($"Unrecognized result: {result}"),
+            };
+            return;
+        }
+
+        private static double Tps(CsharpLoadGeneratorContext context, int requestCount)
+        {
+            return Math.Round((requestCount * 1000.0) / context.StartTime.ElapsedMilliseconds);
+        }
+
+        private static string PercentRequests(CsharpLoadGeneratorContext context, int count)
+        {
+            if (context.GlobalRequestCount == 0)
+            {
+                return "0";
+            }
+            return Math.Round((Convert.ToDouble(count) / context.GlobalRequestCount) * 100.0, 1).ToString();
+        }
+
+        private static string OutputHistogramSummary(HistogramBase histogram)
+        {
+            return $@"
+count: {histogram.TotalCount}
+        p50: {histogram.GetValueAtPercentile(50)}
+        p90: {histogram.GetValueAtPercentile(90)}
+        p99: {histogram.GetValueAtPercentile(99)}
+      p99.9: {histogram.GetValueAtPercentile(99.9)}
+        max: {histogram.GetMaxValue()}
+";
+        }
+    }
+
+
+    internal class Program
+    {
+        static ILoggerFactory InitializeLogging()
+        {
+            return LoggerFactory.Create(builder =>
+                   builder.AddSimpleConsole(options =>
+                   {
+                       options.IncludeScopes = true;
+                       options.SingleLine = true;
+                       options.TimestampFormat = "hh:mm:ss ";
+                   }));
+        }
+
+        const string PERFORMANCE_INFORMATION_MESSAGE = @"
+Thanks for trying out our basic c# load generator!  This tool is
+included to allow you to experiment with performance in your environment
+based on different configurations.  It's very simplistic, and only intended
+to give you a quick way to explore the performance of the Momento client
+running in a dotnet application.
+
+Since performance will be impacted by network latency, you'll get the best
+results if you run on a cloud VM in the same region as your Momento cache.
+
+Check out the configuration settings at the bottom of the 'MomentoLoadGen/Program.cs'
+file to see how different configurations impact performance.
+
+If you have questions or need help experimenting further, please reach out to us!
+        ";
+
+
+        static async Task Main(string[] args)
+        {
+            using ILoggerFactory loggerFactory = InitializeLogging();
+
+            CsharpLoadGeneratorOptions loadGeneratorOptions = new CsharpLoadGeneratorOptions(
+              /**
+               * Configures the Momento client to timeout if a request exceeds this limit.
+               * Momento client default is 5 seconds.
+               */
```

Comment:
Try using triple-front-slash `///` for the C# docstrings

Reply:
thanks gtk